### ISSUE TITLE
printf skiping in my console

### ISSUE THREAD

**Issue by pikakkk** [Internshala-Online-Trainings/programming-with-c-and-cpp-v2#21]
printf statement is not show in my console only scanf is execute 
![Screenshot 2021-08-07 081253](https://user-images.githubusercontent.com/88380813/128597260-f8094cb2-0d4f-4e40-8803-f5d1c4fd7105.png)

**Comment by ghost** [comment]
> printf statement is not show in my console only scanf is execute 
> ![Screenshot 2021-08-07 081253](https://user-images.githubusercontent.com/88380813/128597260-f8094cb2-0d4f-4e40-8803-f5d1c4fd7105.png)
> 

Yeah...eclipse isn't working properly

**Comment by Samrudhipatil51** [comment]
ok sorry for late I hope you understand the why code is not executed if not please let me know

**Comment by ghost** [comment]
> ok sorry for late I hope you understand the why code is not executed if not please let me know

No..i didn't understand why....the code isn't working yet on eclipse
